bxin/ptg2m1m3/m1m3_diagnostic.ipynb

```diff
@@ -6,8 +6,11 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    #    ask Petr to do this.
</pre><p>I think here you now have a method on how to do it, right? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>